src/lib/ketrew_configuration.mli

```diff
@@ -180,3 +142,47 @@ val connection: client -> string
 val token: client -> string
 
 val standalone_of_server: server -> standalone
+
+
+val load_exn:
+  ?and_apply:bool ->
+  ?profile:string ->
+  [ `From_path of string
+  | `Guess
+  | `In_directory of string
+  | `Override of t ] ->
+  t
+(** Load a configuration.
```

Comment:
OK I see what order you meant.

The goal was to hide the side effects, in order to “one day” try to make them less global (which I don't know if it's even possible).